src/app/components/PageLayoutWrapper/index.tsx

```diff
@@ -54,6 +55,7 @@ const PageLayoutWrapper = ({
 }: PropsWithChildren<Props>) => {
   const { service } = useContext(ServiceContext);
   const { isLite, isAmp } = useContext(RequestContext);
+  const isPWA = useIsPWA();
```

Comment:
Will this have any performance issues by using state this high in the render tree?